lldb/include/lldb/Breakpoint/BreakpointOptions.h

```diff
@@ -400,6 +400,12 @@ friend class Breakpoint;
   /// Which options are set at this level.
   /// Drawn from BreakpointOptions::SetOptionsFlags.
   Flags m_set_flags;
+  /// Settings that allow the 'disabled' keyword to be displayed in red.
+  Stream::HighlightSettings m_disbaled_breakpoint_highlight_settings{
+      "disabled",
+      "\x1b[31m",
+      "\x1b[37m",
```

Comment:
You don't need to hardcode these values yourself. Take a look at `AnsiTerminal.h`, there are some constants defined for this purpose.